libs/openFrameworks/gl/ofTexture.cpp

```diff
@@ -1128,35 +1141,50 @@ ofMesh ofTexture::getMeshForSubsection(float x, float y, float z, float w, float
 	// to constantly add a 2 pixel border on all uploaded images
 	// is insane..
 
-	GLfloat offsetw = 0.0f;
-	GLfloat offseth = 0.0f;
+	GLfloat offsetw { 0.0f };
+	GLfloat offseth { 0.0f };
 
 	if (!ofGLSupportsNPOTTextures() && bTexHackEnabled) {
 		offsetw = 1.0f / (texData.tex_w);
 		offseth = 1.0f / (texData.tex_h);
 	}
 	// -------------------------------------------------
 
-	auto topLeft = getCoordFromPoint(sx, sy);
-	auto bottomRight = getCoordFromPoint(sx + sw, sy + sh);
+	auto topLeft { getCoordFromPoint(sx, sy) };
+	auto bottomRight { getCoordFromPoint(sx + sw, sy + sh) };
 
-	GLfloat tx0 = topLeft.x + offsetw;
-	GLfloat ty0 = topLeft.y + offseth;
-	GLfloat tx1 = bottomRight.x - offsetw;
-	GLfloat ty1 = bottomRight.y - offseth;
+	GLfloat tx0 { topLeft.x + offsetw };
+	GLfloat ty0 { topLeft.y + offseth };
+	GLfloat tx1 { bottomRight.x - offsetw };
+	GLfloat ty1 { bottomRight.y - offseth };
 
 	quad.setMode(OF_PRIMITIVE_TRIANGLE_FAN);
 	quad.getVertices().resize(4);
 	quad.getTexCoords().resize(4);
-	quad.getVertices()[0] = {px0,py0,z};
-	quad.getVertices()[1] = {px1,py0,z};
-	quad.getVertices()[2] = {px1,py1,z};
-	quad.getVertices()[3] = {px0,py1,z};
+	
+	quad.getVertices() = {
+		{px0,py0,z},
+		{px1,py0,z},
+		{px1,py1,z},
+		{px0,py1,z},
+	};
 
-	quad.getTexCoords()[0] = {tx0,ty0};
-	quad.getTexCoords()[1] = {tx1,ty0};
-	quad.getTexCoords()[2] = {tx1,ty1};
-	quad.getTexCoords()[3] = {tx0,ty1};
+	quad.getTexCoords() = {
+		{tx0,ty0},
+		{tx1,ty0},
+		{tx1,ty1},
+		{tx0,ty1},
+	};
```

Comment:
to me, this feels like a good use case as it does in one line what is being done in 4 previously.

Reply:
Thanks. I've reverted back others. 
I've changed this one line, I think we get one less increment on shared_ptr this way
```
std::shared_ptr<ofBaseGLRenderer> renderer { ofGetGLRenderer() };
```